commands/db/drop.md

```diff
@@ -3,8 +3,7 @@
 Delete the existing database.
 
 Runs `DROP_DATABASE` SQL statement using `DB_HOST`, `DB_NAME`,
-`DB_USER` and `DB_PASSWORD` database credentials specified in
-wp-config.php.
+`DB_USER` and `DB_PASSWORD` database credentials specified in wp-config.php.
```

Comment:
Well, I'll be brief: all paragraphs starting with a backtick don't work.

Reply:
Fixed in a7d379f